org-formation/650-identity-providers/github-oidc-provider-access.njk

```diff
@@ -66,10 +66,10 @@ Resources:
     Type: 'AWS::IAM::ManagedPolicy'
     Properties:
       PolicyDocument: !Ref PolicyDocument
-{% for Repository in Repositories %}
-  ProviderRole{{ Repository.name | replace('-','') | replace('_','') }}:
+  ProviderRole:
     Type: AWS::IAM::Role
     Properties:
+      RoleName: !Sub '${AWS::StackName}-ProviderRole'
```

Comment:
Set to a specific role name so that we can create dependent PRs to assume
this role for CI purposes.  This will reduce the downtime from deploying
this change.

Reply:
The role name needs to be unique to the account, so this means that we can only deploy this template once per account, right? Should we also add the repository name to the role name?

Reply:
I think it should just update the policy document each time we run.

Reply:
@jesusaurus yes, role names are unique per account.  CFN stack names are also unique.  You can deploy multiple instances as long as the stack name is different.

Reply:
This won't work because IAM role names can only be 64 chars max.  CFN deployment will fail if the name is set to something longer than the max length.  Some of the stack names defined in [650-identity-provider/_task.yaml](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/650-identity-providers/_tasks.yaml) will make the RoleName longer than 64 chars.